.github/workflows/sbt-node-snyk.yml

```diff
@@ -13,6 +13,10 @@ on:
         type: string
         required: false
         default: "11"
+      NODE_VERSION_FILE:
```

Comment:
Does this mean we're able to use different node versions within the same repo? To provide a simple example against this: we've seen a few lambdas with a node 12 runtime, but build with node 14. That's pretty confusing IMO.

Are there any reasons one would want to use a different node version within a single repository?

Reply:
> 1. The repo uses a single node version (defined in `.nvmrc`), but the file is not at the project root (this is my use case - `cdk` projects typically seem to include this file under `/cdk`).

I think GuCDK is wrong here - it should place this file at the root (possibly tricky if it already exists).

> 2. The repo uses a single node version, but it uses [`.node-version`](https://stackoverflow.com/a/27460959/15960596) (rather than `.nvmrc`) to manage the node version.

Not sure we do this ([rough GH search](https://github.com/search?l=&q=org%3Aguardian+.+filename%3A.node-version&type=code)), but a fair point.

> 3. The repo uses multiple node versions e.g. version A for `cdk` and version B for application code. I'm not 100% sure if this is desirable/acceptable and I'm not convinced it'd work properly with this workflow!

See 1. I think we should do everything we can to avoid this as it's pretty confusing!

Reply:
I've merged this PR as I think it's still sensible to allow users to configure this if it's necessary, but I agree with:

> it should place this file at the root (possibly tricky if it already exists).

so I'll see if it can be easily moved for AMIgo.

Reply:
> I've followed the existing convention here (MY_INPUT), but in my experience GitHub Actions inputs are typically defined as my-input (see examples) - I wonder if we should make that change now before too many people pick this up.

Definitely think we should do this sooner rather than later!